turbo.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "start": {
+      "persistent": true,
+      "cache": false,
+      "dependsOn": ["db:start"]
```

Comment:
Questo non dovrebbe dipendere anche da `generate-model`?

Reply:
ah sì, buon punto, l'avevo messo in `build` ma l'avevo dimenticato qui. Aggiungo!